.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
-before_install: "npm install -g npm"
 node_js:
     - "0.10"
-    - "0.11"
+    - "0.12"
```

Comment:
I would leave this the way it is. See https://github.com/mishoo/UglifyJS2/pull/674.